### ISSUE TITLE
Unsupported Lutris version (0.5.17 outdated according to its own starting banner)

### ISSUE THREAD

**Issue by Utopiah** [ivan-hc/AM#1181]
Lutris installation works but while starting it I get a warning that says the current version `0.5.17` is not supported anymore.

**Comment by Utopiah** [comment]
Thanks for the so quick feedback, what's the recommended way in such a case? Not rely on `am`?

**Comment by Samueru-sama** [comment]
> Thanks for the so quick feedback, what's the recommended way in such a case? Not rely on `am`?

Open an issue at the repo of the AppImage.

**Comment by ivan-hc** [comment]
hi, the answer is on top of the README of this repo

https://github.com/ivan-hc/AM#what-doesnt-it-do

**Comment by ivan-hc** [comment]
As far as I know, the developer uses [pkg2appimage](https://github.com/AppImageCommunity/pkg2appimage) to create the AppImage. The recipe is in his repo https://github.com/lucasmz1/Lutris_AppImage/blob/main/lutris.yml

I used to use pkg2appimage myself, and it's definitely not the best way to build a working AppImage.

As you know, I accept products into the database based on user feedback, and this Lutris was suggested to me, @Utopiah you can see https://github.com/ivan-hc/AM/issues/983 , I hesitated a lot before adding it... but if people are okay with it, I'm OK with it. It's not my problem if Lutris or any someone else's package doesn't work.

Of course, if there's something better, let me know. The more reliable a package is, the more likely I'll add it to this list.

**Comment by Utopiah** [comment]
Ah, apologies for the noise created then. I had the suspicion it would be the case (as in a lot of package management situation) but as it's the first time using `am` it happened I was confused. Thanks for sharing the documentation link (which clearly I missed) but even more so for unpacking what happened and how it could potentially be solved.

**Comment by ivan-hc** [comment]
Don't worry.

The fact is that AM manages portable packages, that is, those packages intended to be downloaded directly from the developer's repository.

That said, I have no power over the sources. We don't have servers where we collect actual packages, but scripts that intercept and download them. As the AUR does. That's all.

The only power I have here is to accept/upload installation scripts upon user request.

For my part, I always prefer to upload official programs, and I always try to push third-party packagers to submit their packages to the uptream, in order to push those in charge to create their own, officially.

I also create AppImage packages, and as the upstreams decide to release official ones, I archive/close my repositories.

If then, upon request, someone asks me to upload something unofficial but that they trust, I do it. I leave the decision to my users. And always through feedback, I can push them to improve other projects... or they can propose something better to me.

**Comment by lucasmz1** [comment]
Hello, I've used appimagebuilder, and now is updated for the latest version 0.5.18. but yet I couldn't fix some pixmaps issues in others linux distros such as slackware. python gtk appas are such a complex thing. in ubuntu it was working fine, at least lubuntu 24.04 as I tested myself.
wish somebody could give me some tips about gtk apps. Thanks again guys add my application in your software repo and list.
